politeiawww/cmd/shared/client.go

```diff
@@ -1134,8 +1133,8 @@ func (c *Client) GetAllVetted(gav *www.GetAllVetted) (*www.GetAllVettedReply, er
 	return &gavr, nil
 }
 
-// NewComment submits a new proposal comment for the logged in user.
-func (c *Client) NewComment(nc *www.NewComment) (*www.NewCommentReply, error) {
+// CmsNewComment submits a new proposal comment for the logged in user.
+func (c *Client) CmsNewComment(nc *www.NewComment) (*www.NewCommentReply, error) {
```

Comment:
Should be www new comment, not cms new comment since its a route.  CMS just happens to use the www route, but the function should be named based on the route, not what is calling it.